rcd_fishhook/rcd_fishhook.c

```diff
@@ -119,7 +119,9 @@ static void rcd_perform_rebinding_with_section(struct rcd_rebindings_entry *rebi
               indirect_symbol_bindings[i] != cur->rebindings[j].replacement) {
             *(cur->rebindings[j].replaced) = indirect_symbol_bindings[i];
           }
-          indirect_symbol_bindings[i] = cur->rebindings[j].replacement;
+          if (i < (sizeof(indirect_symbol_bindings) / sizeof(indirect_symbol_bindings[0]))) {
```

Comment:
I'm confused how this works. Sorry if I'm being dumb. sizeof(indirect_symbol_bindings) is sizeof(void**) and sizeof(indirect_symbol_bindings[0]) is sizeof(void*) so this is equivalent to if (i < 1) isn't it?

Reply:
I expect `-Wsizeof-pointer-div` would flag this for you?